features/verblijfstitel.feature

```diff
@@ -2,38 +2,46 @@
 
     @gba
     Scenario: verblijfstitel heeft geen datum einde
-      Gegeven de persoon met burgerservicenummer '555550001' heeft de volgende 'verblijfstitel' gegevens
+      Gegeven de persoon met burgerservicenummer '000000012' heeft de volgende 'verblijfstitel' gegevens
       | aanduiding verblijfstitel (39.10) | datum ingang verblijfstitel (39.30) |
       | 37                                | 20210315                            |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer | 555550001                       |
-      | fields              | verblijfstitel.aanduiding.code  |
+      | burgerservicenummer | 000000012                       |
+      | fields              | verblijfstitel.aanduiding       |
       Dan heeft de response een persoon met alleen de volgende 'verblijfstitel' gegevens
-      | naam            | waarde |
-      | aanduiding.code | 37     |
+      | naam                    | waarde                                                                              |
+      | aanduiding.code         | 37                                                                                  |
+      | aanduiding.omschrijving | code Vw 2000 art. 8, onder e, gemeenschapsonderdaan econ. niet-actief, arbeid spec. |
 
     @gba
     Abstract Scenario: verblijfstitel heeft <titel>
-      Gegeven de persoon met burgerservicenummer '555550001' heeft de volgende 'verblijfstitel' gegevens
+      Gegeven de persoon met burgerservicenummer '000000024' heeft de volgende 'verblijfstitel' gegevens
       | aanduiding verblijfstitel (39.10) | datum einde verblijfstitel (39.20) | datum ingang verblijfstitel (39.30) |
       | 37                                | <datumEinde>                       | 20210315                            |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer | 555550001                       |
+      | burgerservicenummer | 000000024                       |
       | fields              | verblijfstitel.aanduiding.code  |
       Dan heeft de response een persoon met alleen de volgende 'verblijfstitel' gegevens
-      | naam            | waarde |
-      | aanduiding.code | 37     |
+      | naam                    | waarde                                                                              |
+      | aanduiding.code         | 37                                                                                  |
+      | aanduiding.omschrijving | code Vw 2000 art. 8, onder e, gemeenschapsonderdaan econ. niet-actief, arbeid spec. |
 
       Voorbeelden:
```

Comment:
voorbeeld "datum einde is morgen" moet net als volgend scenario een dynamische waarde <morgen> krijgen.

Hoe kon dit scenario slagen? De datum die er nu staat (20220606) ligt in het verleden, dus deze test had moeten falen! Heb je dit scenario nog niet uitgevoerd of is de automation niet correct?